src/components/inputBox.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import { mediaQuery, calcSize } from 'styles/media';
+import color from 'styles/colors';
+
+interface InputBoxProps extends React.InputHTMLAttributes<HTMLInputElement> {
+  label: string | undefined;
```

Comment:
```suggestion
  label?: string;
```
요렇게 작성하면 좀 더 깔끔하지 않을까요?